server/src/test/java/io/spine/examples/kanban/server/board/BoardTest.java

```diff
@@ -129,6 +131,180 @@ void rejection() {
         }
     }
 
+    @Nested
+    class MoveColumns {
+
+        @Test
+        @DisplayName("move columns left")
+        void moveLeft() {
+            ColumnPosition from = ColumnPositions.of(1, 4);
+            ColumnId column = currentBoardState().getColumn(from.zeroBasedIndex());
+            ColumnPosition to = ColumnPositions.of(4, 4);
+
+            context().receivesCommand(moveColumn(column, from, to));
+
+            ColumnId actual = currentBoardState().getColumn(to.zeroBasedIndex());
+            assertThat(column).isEqualTo(actual);
+        }
+
+        private Board currentBoardState() {
+            return (Board) context()
+                    .assertEntityWithState(board(), Board.class)
+                    .actual()
+                    .state();
+        }
+
+        private MoveColumn moveColumn(ColumnId column, ColumnPosition from, ColumnPosition to) {
+            return MoveColumn
+                    .newBuilder()
+                    .setColumn(column)
+                    .setBoard(board())
+                    .setFrom(from)
+                    .setTo(to)
+                    .vBuild();
+        }
+
+        @Test
+        @DisplayName("move columns right")
+        void moveRight() {
+            ColumnPosition from = ColumnPositions.of(4, 4);
+            ColumnId column = currentBoardState().getColumn(from.zeroBasedIndex());
+            ColumnPosition to = ColumnPositions.of(1, 4);
+
+            context().receivesCommand(moveColumn(column, from, to));
+
+            ColumnId actual = currentBoardState().getColumn(to.zeroBasedIndex());
+            assertThat(column).isEqualTo(actual);
+        }
+
+        @Test
+        @DisplayName("reject moving columns if `from` is invalid")
+        void invalidFrom() {
+            ColumnPosition invalidFrom = invalidPosition();
+            ColumnId column = ColumnId.generate();
+            ColumnPosition to = ColumnPositions.of(1, 4);
+
+            context().receivesCommand(moveColumn(column, invalidFrom, to));
+
+            ColumnCannotBeMoved expected =
+                    ColumnCannotBeMoved.newBuilder()
```

Comment:
The creation of the messages of this type is repeated for several times. Please extract within this class.